src/modules/sdlog2/sdlog2.c

```diff
@@ -72,6 +72,7 @@
 #include <uORB/topics/actuator_outputs.h>
 #include <uORB/topics/actuator_controls.h>
 #include <uORB/topics/vehicle_command.h>
+#include <uORB/topics/vehicle_land_detected.h>
```

Comment:
Ok, I will merge both land detection modules into one land detector and run different algorithms depending on the underlying system, e.g: "land_detector start fixedwing"

Ah, I did not think about STAT message already containing land. Then LAND is indeed redundant and can be removed.